deployment/CP05MOAS-GL340_Deploy.csv

```diff
@@ -53,9 +53,9 @@ AR-39,,AR-40,,CP05MOAS-GL340-03-CTDGVM000,9,1,2019-09-27T12:26:00,2019-11-05T15:
 AR-39,,AR-40,,CP05MOAS-GL340-04-DOSTAM000,9,1,2019-09-27T12:26:00,2019-11-05T15:00:21,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00398,40.0501,-70.6002,,200,200,EB
 AR-39,,AR-40,,CP05MOAS-GL340-05-PARADM000,9,1,2019-09-27T12:26:00,2019-11-05T15:00:21,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50145,40.0501,-70.6002,,200,200,EB
 AR-39,,AR-40,,CP05MOAS-GL340-01-ADCPAM000,9,1,2019-09-27T12:26:00,2019-11-05T15:00:21,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-48878,40.0501,-70.6002,,200,200,EB 
-TS20191212,,,,CP05MOAS-GL340-00-ENG000000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGCON-GLDRCP-00340,40.0817,-70.74813,,200,200,EB 
-TS20191212,,,,CP05MOAS-GL340-02-FLORTM000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-FLORTM-03921,40.0817,-70.74813,,200,200,EB 
-TS20191212,,,,CP05MOAS-GL340-03-CTDGVM000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-CTDGVM-09029,40.0817,-70.74813,,200,200,EB 
-TS20191212,,,,CP05MOAS-GL340-04-DOSTAM000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00398,40.0817,-70.74813,,200,200,EB 
-TS20191212,,,,CP05MOAS-GL340-05-PARADM000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50145,40.0817,-70.74813,,200,200,EB 
-TS20191212,,,,CP05MOAS-GL340-01-ADCPAM000,10,1,2019-12-13T01:04:47,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-48878,40.0817,-70.74813,,200,200,EB 
\ No newline at end of file
+TS20191212,,AR42-02,,CP05MOAS-GL340-00-ENG000000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGCON-GLDRCP-00340,40.0817,-70.74813,,200,200,EB 
+TS20191212,,AR42-02,,CP05MOAS-GL340-02-FLORTM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-FLORTM-03921,40.0817,-70.74813,,200,200,EB 
+TS20191212,,AR42-02,,CP05MOAS-GL340-03-CTDGVM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-CTDGVM-09029,40.0817,-70.74813,,200,200,EB 
+TS20191212,,AR42-02,,CP05MOAS-GL340-04-DOSTAM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00398,40.0817,-70.74813,,200,200,EB 
+TS20191212,,AR42-02,,CP05MOAS-GL340-05-PARADM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50145,40.0817,-70.74813,,200,200,EB 
+TS20191212,,AR42-02,,CP05MOAS-GL340-01-ADCPAM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-48878,40.0817,-70.74813,,200,200,EB 
```

Comment:
looks good - not sure where I can confirm date/time since this isn't on QLR but it fits with update you gave